node/cmd/guardiand/logging.go

```diff
@@ -0,0 +1,28 @@
+package guardiand
+
+import (
+	"go.uber.org/zap/buffer"
+	"go.uber.org/zap/zapcore"
+	"unicode"
+)
+
+type consoleEncoder struct {
+	zapcore.Encoder
+}
+
+func (e consoleEncoder) EncodeEntry(entry zapcore.Entry, fields []zapcore.Field) (*buffer.Buffer, error) {
+	buf, err := e.Encoder.EncodeEntry(entry, fields)
+	if err != nil {
+		buf.Free()
+		return nil, err
+	}
+
+	b := buf.Bytes()
+	for i := range b {
+		if unicode.IsControl(rune(b[i])) && !unicode.IsSpace(rune(b[i])) {
+			b[i] = '\x1A' // Substitute character
+		}
+	}
+
+	return buf, nil
+}
```

Comment:
This exact same file with the exact same contents is already in #744. I'm sure git will dedup, but was this intentional?